### DISCUSSION TITLE
BETA Client - Switching between themes like current client

### DISCUSSION THREAD

**Discussion by lucina-da-lucario** [stoatchat/discussions#782] [Feature Suggestions]
Short and to the point, I tried out the Beta a little after seeing a link to it in another discussion thread here. And my eyes got nuked by the bright pink colors. Any chance the feature to switch themes will get added to the Beta, or at the very least, a Dark Mode? I couldn't find an option for it as it is now, and this is genuinely an eyesore.
![Screenshot_20241002_221425_Firefox](https://github.com/user-attachments/assets/1fac782a-4065-464e-9230-0b4b944a1571)
![Screenshot_20241002_221807_Firefox](https://github.com/user-attachments/assets/2b45bdba-f489-4734-9f90-fb2a0f1e9a02)

**Comment by amycatgirl** [comment]
You can switch themes on the login page by clicking the moon icon, or by opening your browser's console (ctrl+shift+I, then click the console tab) and typing `__demo_setDarkMode(true)` (if I remember correctly)

Since you are using a mobile phone I would recommend you to clear the site's storage by going into Chrome settings > Site settings > All sites > revolt.chat, then click "Delete data and reset permissions". Finally, on the login page, click the moon icon to switch themes.

Hope this helps! 😄

\- Amy

**Comment by lucina-da-lucario** [comment]
Oh, I'm on Firefox, actually. Still, good to know. I probably won't go to the Beta again, though. I was more curious about it than anything. (That, and for some reason, the link's giving me a blank page after showing a frame of the login screen. Not sure why.) Thanks, anyway.
